qa/rpc-tests/test_framework/util.py

```diff
@@ -312,12 +312,12 @@ def start_node(i, dirname, extra_args=None, rpchost=None, timewait=None, binary=
     if binary is None:
         binary = os.getenv("ZCOIND", "zcoind")
         print(binary)
-    args = [ binary, "-datadir="+datadir, "-server", "-keypool=1", "-discover=0", "-rest", "-reindex" ,"-mocktime="+str(get_mocktime()) ]
+    args = [ binary, "-datadir="+datadir, "-server", "-keypool=1", "-discover=0", "-rest", "-reindex", "-dandelion=0", "-mocktime="+str(get_mocktime()) ]
```

Comment:
need to support enable dandelion because have test file `p2p-dandelion.py`

Reply:
I think, that now it is ok, cause the test still commented

Reply:
It's not a big deal. then I'll create a card at backlog to support dandelion test.